model_servers/llamacpp_python/README.md

```diff
@@ -88,7 +90,8 @@ Place all models in the [models](../../models/) directory.
 You can use this snippet below to download the default model:
 
 ```bash
-make -f Makefile download-model-granite
+cd ../../models
+make download-model-granite
```

Comment:
Both of these are technically correct, what you have and what I had previously. This is because I have an [override command in the makefile](https://github.com/containers/ai-lab-recipes/blob/main/model_servers/llamacpp_python/Makefile#L28-L31) which will call that make command in models. You can change this if you want but I think it is more correct to use the override.

Reply:
I did not notice that in the llamacpp_python/Makefile, updated!